.eslintrc.js

```diff
@@ -0,0 +1,19 @@
+module.exports = {
```

Comment:
In the frontend we use this

```js
module.exports = {
  root: true,
  parserOptions: {
    ecmaVersion: 2017,
    sourceType: 'module'
  },
  extends: 'eslint:recommended',
  env: {
    browser: true,
    es6: true
  },
  rules: {
    "arrow-parens": ["error", "as-needed"],
    "comma-dangle": ["error", "never"],
    "no-console": ["warn", { allow: ["error"] }]
  }
};
```

Reply:
I'm ok with these - I added the `env`s (+ a `node` one as well), as well as `no-console` and changed `arrow-parens`.